include/enotify.php

```diff
@@ -624,6 +624,11 @@ function check_item_notification($itemid, $uid, $notification_type) {
 		return false;
 	}
 
+	if (DI::pConfig()->get(local_user(), 'system', 'notify_ignored', true) && Contact\User::isIgnored($item['author-id'], $uid)) {
```

Comment:
Shouldn't the test be `!DI::pConfig()->get(local_user(), 'system', 'notify_ignored', true)` since showing the notifications for ignored contacts would skip this condition?

Reply:
Yeah, right!

Reply:
BTW: It would be great having a default file for the pconfig settings as well.